src/components/Banner/Banner.module.css

```diff
@@ -156,15 +157,17 @@
 \*------------------------------------*/
 
 /**
- * Flat banner
- * 1) Has no drop shadow or visible border so it sits flat on the page
- * 2) Instead of removing the border, we replace it with a transparent one.
- *    On windows high contrast mode, the background color will be removed,
- *    but this border will become 100% opacity black to retain visual separation
- * 3) Remove the bottom margin which is usually there to account for the drop shadow length
+ * Flat banner.
  */
 .banner--flat {
-  box-shadow: none; /* 1 */
-  --messaging-border-color: transparent; /* 2 */
-  margin-bottom: 0; /* 3 */
+  /* Has no drop shadow or visible border so it sits flat on the page */
+  box-shadow: none; 
+  /**
+  * Instead of removing the border, we replace it with a transparent one.
+  * On windows high contrast mode, the background color will be removed,
+  * but this border will become 100% opacity black to retain visual separation
+  */
```

Comment:
```suggestion
   * Instead of removing the border, we replace it with a transparent one.
   * On windows high contrast mode, the background color will be removed,
   * but this border will become 100% opacity black to retain visual separation
   */
```